go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	github.com/moby/ipvs v1.1.0
 	github.com/mrunalp/fileutils v0.5.1
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822
-	github.com/onsi/ginkgo/v2 v2.17.2
+	github.com/onsi/ginkgo/v2 v2.6.1-0.20241008152707-25bf9f14db44
```

Comment:
Do you mind leaving this commit removing `go.*` changes from this (`UPSTREAM: <carry>: add openshift-tests-extension compatible binary`)? This will help a lot with the next rebase.

Reply:
So put any of the go.* changes in the `<drop>` commit?